addons/website/views/snippets/s_mockup_image.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_mockup_image" name="Mockup Image">
+    <section class="s_mockup_image o_cc o_cc1 pt72 pb72" data-oe-shape-data="{'shape':'web_editor/Wavy/11', 'colors':{'c1': '#2D3142', 'c4': '#FFFFFF'}, 'showOnMobile':true}">
+        <div class="o_we_shape o_web_editor_Wavy_11 o_shape_show_mobile" style="background-image: url(&quot;/web_editor/shape/web_editor%2FWavy%2F11.svg?c1=%232D3142&amp;c4=%23FFFFFF&quot;);"/>
+        <div class="container">
+            <div class="row align-items-center">
+                <div class="col-lg-4 pt16 pb16">
+                    <h2>All your business on one platform.</h2>
```

Comment:
Remember to adapt the heading text accordingly to the image that you'll choose (the mockup one was not bad IMO)

<img width="814" alt="image" src="https://github.com/user-attachments/assets/60933544-82b2-43da-b69f-9988760ed022">

Reply:
All fixed